app/models/user.rb

```diff
@@ -146,6 +146,29 @@ def self.sorted_observer_response_counts(mission, limit)
       LIMIT ?", mission.id, mission.id, limit]).reverse
   end
 
+  # Returns all non-admin users in the form's mission with the given role that have
+  # not submitted any responses to the form
+  #
+  # options[:role]) the role to check for
+  # options[:limit]) how many users we want to fetch from the db
+  def self.without_responses_for_form(form, options)
+    find_by_sql(["SELECT SQL_CALC_FOUND_ROWS * FROM users
+                  INNER JOIN assignments WHERE
+                  assignments.user_id = users.id AND
+                  assignments.mission_id = ? AND
+                  assignments.role = ? AND
+                  users.admin = FALSE AND
+                  users.id NOT IN (
+                    SELECT distinct users.id FROM users
+                    INNER JOIN responses WHERE
+                    responses.user_id = users.id AND
```

Comment:
Same here. See this answer: http://stackoverflow.com/questions/4160637/mysql-join-clause-vs-where-clause